tools/pipelines/templates/include-install-build-tools.yml

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# include-install-build-tools
+#
+# This template can be included in pipelines to install the Fluid build-tools locally so flub can be used in the
+# pipeline.
+
+parameters:
+- name: buildDirectory
+  type: string
+
+- name: buildToolsVersionToInstall
+  type: string
+  default: repo
+
+# The path to the pnpm store.
+- name: pnpmStorePath
+  type: string
+  default: $(Pipeline.Workspace)/.pnpm-store
+
+steps:
+
+  # These steps should ONLY run if we're using the repo version of the build tools. These steps are mutually exclusive
+  # with the next group of steps.
+  - ${{ if eq(parameters.buildToolsVersionToInstall, 'repo') }}:
+    - task: Bash@3
+      name: PrependPath
+      displayName: Prepend build-tools CLI to path
+      inputs:
+        targetType: 'inline'
+        workingDirectory: ${{ parameters.buildDirectory }}
+        script: |
+          # Prepend the cli bin dir to the path. See
+          # <https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#prependpath-prepend-a-path-to-the--path-environment-variable>
+          # more information.
+          echo "##vso[task.prependpath]$(Build.SourcesDirectory)/build-tools/packages/build-cli/bin"
```

Comment:
This bit isn't here in the rc5 PR, is that expected?

Reply:
Was this change already in RC5?

Reply:
This bit is present in rc4 but not in rc5.